projection/src/test/scala/akka/projection/r2dbc/R2dbcProjectionSpec.scala

```diff
@@ -1295,7 +1295,6 @@ class R2dbcProjectionSpec
     }
 
     "pause projection" in {
-      pending // FIXME not implemented yet
```

Comment:
there are a few more pending in R2dbcOffsetStoreSpec, R2dbcTimestampOffsetStoreSpec, R2dbcTimestampOffsetProjectionSpec

Reply:
`pending` removed, but not in `R2dbcTimestampOffsetProjectionSpec`. 

There we have another related with how we read the current offset. We don't take fill in the `readTimestamp` and it's always EPOCH causing the test to fail. I think we should fix this in another PR.